### DISCUSSION TITLE
Support for android

### DISCUSSION THREAD

**Discussion by goodnewline** [pot-app/pot-desktop#559] [Q&A]
When will there be a release for Android?

**Comment by Pylogmon** [comment]
Yes, I have wanted to support mobile for a long time, but until now, I have only created a repository [pot-mobile](https://github.com/pot-app/pot-mobile) and I have no time to complete it because I know nothing about mobile development, so the release of the Android version may take a long time, At least until [Tauri](https://github.com/tauri-apps/tauri) officially releases version 2.0, I will consider using Tauri to implement it. Until then, please stay tuned!
